docs/workflow.md

```diff
@@ -333,7 +340,7 @@ process foo {
     result = "$data world"
 }
 
-process bar {
+process toUpper {
```

Comment:
I don't have strong opinions, but for some examples we use snake case and some we use camel case. Do you think we could standardize?

Reply:
Actually this looks like a quick fix, I'll go ahead and do it